docs/discover/search.asciidoc

```diff
@@ -90,65 +88,72 @@ used for the saved search will also be automatically selected.
 
 [[save-load-delete-query]]
 === Saving and Loading Queries
-Saving queries enables you to reload them into <<discover, Discover>>, <<visualize, visualizations>> and <<dashboard, dashboards>>.
-Saving a query saves the query string, the filters and the time filter but, unlike searches, the index pattern is not saved, making saved queries portable.
+Saving queries allows you to reuse them in <<discover, Discover>>, <<visualize, visualizations>> and <<dashboard, dashboards>>.
+Saved queries always include the KQL or Lucene query text, but the filter bar and time picker can optionally be included as well. Unlike saved searches, queries do not include Discover specific information like the columns selected in the document table, the sort order, or the index pattern. Saved searches can be thought of as a way to persist a particular Discover view. Saved queries on the other hand are a portable set of filters that can be reused in other applications.
 
 ==== Read only access
```

Comment:
Was there an x-pack label next to this before or am imagining things? I see there is one in the saved search section so I'm not sure if that's what I was seeing before or not. Probably we should have one here since there is one in the saved search section?

Reply:
I had one there but then removed it. I'll put it back.

Reply:
https://github.com/elastic/kibana/pull/43731/commits/1924aaebb70e917ca6dc5682ea2af9fcb8391d85#diff-87a4343c7de08863ed7fa9d869b5c5f1R94